textile/chat-features.textile

```diff
@@ -413,12 +416,16 @@ Typing Indicators allow chat room users to indicate to others that they are typi
 * @(CHA-T3)@ @[Testable]@ Users may configure a timeout interval for when they are typing. This configuration is provided as part of the @RoomOptions@ @typing.timeoutMs@ property, or idiomatic equivalent. The default is 5000ms.
 * @(CHA-T4)@ Users may indicate that they have started typing.
 ** @(CHA-T4a)@ @[Testable]@ If typing is not already in progress, per explicit cancellation or the timeout interval in (@CHA-T3@), then a new typing session is started.
-*** @(CHA-T4a1)@ @[Testable]@ When a typing session is started, the client is entered into presence on the typing channel.
```

Comment:
We should mark `CHA-T4a1` as removed, instead it's redefined at the bottom. Is this acceptable?

Reply:
There's not any change in functionality (beyond what was originally intended) - so its just a rewording :)